packages/core/src/payment/strategies/stripe-upe/stripe-upe-payment-strategy.ts

```diff
@@ -34,6 +34,7 @@ export default class StripeUPEPaymentStrategy implements PaymentStrategy {
     private _stripeElements?: StripeElements;
     private _isMounted = false;
     private _unsubscribe?: (() => void);
+    private _isDesinitialize?: boolean;
```

Comment:
```suggestion
    private _isDeinitialize?: boolean;
```

same for the other places

Reply:
Hi Animesh, I alredy do the changes, can you do the validation again, please?